extensions/amp-form/0.1/amp-form.js

```diff
@@ -189,10 +267,17 @@ function installSubmissionHandlers(win) {
 }
 
 
+/**
+ * @param {!Window} win
+ * @private
+ */
 function installAmpForm(win) {
   return getService(win, 'amp-form', () => {
     if (isExperimentOn(win, TAG)) {
-      installStyles(win.document, CSS, () => installSubmissionHandlers(win));
+      installStyles(win.document, CSS, () => {
+        validationBubble = new ValidationBubble(window);
```

Comment:
I'm not sure I like having a global `validationBubble`. Why not just make it an instance variable on `AmpForm`? What if there are multiple forms on the page?